jupyterlab/labbox_ephys_widgets_jp/labbox_ephys_widgets_jp/create_sorting_view.py

```diff
@@ -73,6 +84,18 @@ def set_external_unit_metrics(self, external_unit_metrics):
         def _handle_message(self, widget, msg, buffers):
             if msg['type'] == 'iterate':
                 self._worker_session.iterate()
+            elif msg['type'] == 'appendSubfeedMessage':
```

Comment:
We need ability to append subfeed messages and add subfeed watches (to watch for changes) from the browser